src/OrbitGl/TimeGraph.cpp

```diff
@@ -1,9 +1,8 @@
+#include "TimeGraph.h"
```

Comment:
I assume clang-format or ide messed up somehow

Reply:
Weird, probably that was due to Renaming in VS. Thanks for spotting, fixed it.